main.js

```diff
@@ -0,0 +1,99 @@
+
+const searchButton = document.getElementById('button');
+searchButton.addEventListener('click', getCountryInfo);
+
+const searchBar = document.getElementById('textbar');
+searchBar.addEventListener('keyup', setQuery);
+
+const countryInfoBox = document.getElementById('countries');
+
+let query = '';
+
+function setQuery(e) {
+    query = e.target.value;
+    if (e.keyCode === 13) {
```

Comment:
`keyCode` is deprecated. Gebruik daarom `key`. Bron: https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode